Makefile

```diff
@@ -1,6 +1,14 @@
+dev: check-env
+	cd .. && pip3 uninstall -y sdx-common && pip3 install -I ./sdx-common
```

Comment:
This will break if sdx-common isn't already installed. See downstream-ctp for a fix.